packages/swap/src/processBlocks.ts

```diff
@@ -53,7 +53,12 @@ const fetchBlocks = async (height: number): Promise<Block[]> => {
   throw new Error('failed to fetch batch');
 };
 
-export default async function processBlocks() {
+export default async function processBlocks(
+  preBlockHook?: (
```

Comment:
why are we passing this as an argument? should be able to just import this right?

Reply:
was just imagining the inevitable scenario of swapping app ephemeral processors. i've inlined it for now